pages/en/contrib/doc-contrib.md

```diff
@@ -25,13 +25,12 @@ Follow these steps:
 1. Click on **Edit this page** (top of page) to fork the repo.
 1. Edit the page or pages as needed.
 1. Commit to your fork/branch.
-1. Open a PR for your changes.  If there is an associated issue, please reference it in your PR.
+1. Open a PR for your changes.  If there is an associated issue, please reference it in your PR. Check out [Submitting a pull request to LoopBack 4](https://loopback.io/doc/en/lb4/submitting_a_pr.html) for detailed instructions.
 
 {% include important.html content="When you submit your PR, be sure to agree to the contributor license agreement; see below.
 " %}
 
-We attempt to review and merge PRs as soon as possible; in general, we'll try to do so within 24 hours during the business week.  Please allow longer over weekends or holidays.
-
+We attempt to review and merge PRs as soon as possible; in general, we'll try to do so within 24 hours during the business week.  Please allow longer over weekends or holidays.                
```

Comment:
not sure why it's showing there's a change here.  I think we still want to keep the extra line.  Perhaps @raymondfeng meant [this  comment](https://github.com/strongloop/loopback.io/pull/816#discussion_r260487590) is not to have extra space in that new line.